ember/app/ui/components/dynamic-table/template.hbs

```diff
@@ -7,8 +7,19 @@
   <thead>
     <tr>
       {{#each (map-by "heading" @config.columns) as |header|}}
-        <th class={{join " " header.classList}}>
-          {{t header.label}}
+        <th
+          class={{join " " header.classList}}
+        >
+          {{#if header.sortKey}}
+            <UkButton @color="text" @onClick={{fn this.updateOrder header.sortKey}}>
+              {{t header.label}}
+              {{#if (eq this.orderKey header.sortKey)}}
+                <span uk-icon="icon: {{this.sortIcon}}"></span>
+              {{/if}}
+            </UkButton>
+          {{else}}
+            {{t header.label}}
+          {{/if}}
```

Comment:
Is this formatted with prettier? To me it seems like the lines are longer than normal prettier standards.

Reply:
Ran prettier over the file